src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/PipelineTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using Azure.Core.Http;
+using Azure.Core.Http.Pipeline;
+using Azure.Core.Testing;
+using NUnit.Framework;
+using System;
+using System.Diagnostics.Tracing;
+
+namespace Azure.Core.Tests
+{
+    // TODO (pri 2): Do use the EventRegister NuGet package or the standalone eventRegister.exe tool, to run build-time validation of the event source classes defined in your assemblies.
```

Comment:
What should we do with all the TODOs?

Reply:
I will scrub the code and put all of them in the issue that is tracking the TODO (https://github.com/Azure/azure-sdk-for-net-lab/issues/38)

Reply:
I think I will keep the Todos in the code -- I think it is easier for whoever is fixing the todos to understand what the todo is referring to. I have put the todos in an issue and we will start fixing the issues there.